core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -321,9 +322,8 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
 
         final String hostname = hostname(headers);
         final VirtualHost host = config.findVirtualHost(hostname);
-
         final RoutingContext routingCtx =
-                DefaultRoutingContext.of(host, hostname, pathAndQuery.path(), pathAndQuery.query(),
+                DefaultRoutingContext.of(host, hostname, UUID.randomUUID(), pathAndQuery.path(), pathAndQuery.query(),
```

Comment:
I think we don't need this `uuid` for `RoutingContext`. If we need that we have to use the one from the `requestLog`. WDYT? /cc @trustin, @ikhoon

Reply:
Agreed.